src/main/java/org/example/attorney/AttorneyScrapper.java

```diff
@@ -0,0 +1,20 @@
+package org.example.attorney;
+
+import org.jsoup.*;
+import org.jsoup.nodes.*;
+import org.jsoup.select.*;
+
+import java.io.*;
+import java.util.*;
+
+public class AttorneyScrapper {
+    public static List<String> getAttorneyLinks(String baseUrl) throws IOException {
```

Comment:
using static on methods is procedural style not OOP
```suggestion
    public List<String> getAttorneyLinks(String baseUrl) throws IOException {
```